### PR TITLE
Fix vixenNetwork scraper

### PR DESCRIPTION
**Pull request by bnkai**
Now uses the graphql endpoint for all sites

### CODE REVIEW

File: scrapers/vixenNetwork.py [hunk 1]
```diff
@@ -1,126 +1,150 @@
...
+        Site('Tushy'),
+        Site('Tushy Raw'),
+        Site('Slayed'),
+        Site('Vixen')
+}
+
+frag = json.loads(sys.stdin.read())
+url = frag.get("url")
+if url is None:
+  log.error(f"No URL given")
```
Comment (Belleyy) [inline scrapers/vixenNetwork.py:137]: don't need the `f`

Comment (Belleyy) [reply scrapers/vixenNetwork.py:137]: Just thinking that the whole block `if url is None:` is useless...
You can't use this scraper if you don't put a URL so this condition will never happens.

**Comment by bnkai** [comment]
scraper update with code by @Belleyy to support scrapeByName

for the sceneByName
 If your search query contains the studio name, the scraper will only search on that site.
`Ariana Vixen Cecilia` will only search on Vixen for `Ariana Cecilia`
 If your search query starts with a '$' all sites will be searched
`'$Ariana Vixen Cecilia` will search in all sites for `Ariana Vixen Cecilia`

File: scrapers/vixenNetwork.py [hunk 2]
```diff
@@ -1,126 +1,298 @@
...
+
+    #  Only search on specific site if the studio name is in the search query
+    # ('Ariana Vixen Cecilia' will search only on Vixen)
+
+    # if the first character is $, filter will be ignored.
+    if search_query[0] != "$":
+        # make sure longer matches are filtered first
+        studios_sorted = sortByLength(studios)
+        for x in studios_sorted:
+            if x.id.lower() not in search_query:
```
Comment (Belleyy) [inline scrapers/vixenNetwork.py:274]:
```suggestion
            if x.id.lower() in search_query:
```

File: scrapers/vixenNetwork.py [hunk 3]
```diff
@@ -1,126 +1,298 @@
...
+        search_query = search_query[1:]
+
+    if filter:
+        log.info(f"Filter: {filter} applied")
+
+    log.debug(f"Query: '{search_query}'")
+
+    for x in studios:
+        if filter:
+            if x.id.lower() in filter:
```
Comment (Belleyy) [inline scrapers/vixenNetwork.py:289]:
```suggestion
            if x.id.lower() not in filter:
```

**Comment by tolotos68** [comment]
Hi, just a question - the scraper doesn't get any tags, right? Is there a chance of using the member's section (if one has an account)?

**Pull request merged**